packages/core/helpers/fetchRemoteData.js

```diff
@@ -1,44 +1,39 @@
 import Papa from 'papaparse'
-import { isSolrCsv, isSolrJson } from '@cdc/core/helpers/isSolr'
+import { isSolrCsv } from '@cdc/core/helpers/isSolr'
 
-export default async function (_url, visualizationType = '') {
+export default function fetchRemoteData(_url) {
```

Comment:
@adamdoe making this whole function async meant that it returns either a promise or a rejected promise. That means in order to handle the error which comes out of this you have to wrap any fetchRemoteData function call with try catch. Rather than doing that I refactored this to a common promise. I'm not sure if this could cause some issues in other components but now I am able to handle errors in a traditional .catch chain. as you see in the CdcDashboardComponent. 
 
 Without this there's no way to handle a 500 error within CdcDashboard.